src/Steppers/Steppers.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { FC } from 'react';
+
+import classNames from '../utils/classNames';
+
+import './steppers.scss';
+
+export type Step = {
+    title: string;
+    caption?: string;
+    active?: boolean;
+    success?: boolean;
+    warn?: boolean;
+    fail?: boolean;
+};
+
+export interface Props {
+    title?: string;
+    steps: Step[];
+}
+
+const Steppers: FC<Props> = ({ title, steps }) => (
```

Comment:
```suggestion
export interface Steppers {
    title?: string;
    steps: Step[];
}

const Steppers = ({ title, steps }: Steppers) => (
```

Reply:
If I'm not mistaken, we usually just add the types of the props in the parameters. And typescript has a cool feature that lets the type definition live together with the value-definition. Meaning that both the `type Steppers` and the functional component `const Steppers` can live together.

So if you choose to use this approach, remember to remove the `{ FC }` dependency as well.